compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/TooltipArea.desktop.kt

```diff
@@ -135,19 +135,21 @@ fun TooltipArea(
             .onGloballyPositioned { parentBounds = it.boundsInWindow() }
             .onPointerEvent(PointerEventType.Enter) {
                 cursorPosition = it.position
-                startShowing()
+                if (!isVisible && !it.buttons.areAnyPressed) {
+                    startShowing()
+                }
             }
             .onPointerEvent(PointerEventType.Move) {
                 cursorPosition = it.position
-                hideIfNotHovered(parentBounds.topLeft + it.position)
+                if (!isVisible && !it.buttons.areAnyPressed) {
+                    startShowing()
+                }
             }
             .onPointerEvent(PointerEventType.Exit) {
                 hideIfNotHovered(parentBounds.topLeft + it.position)
             }
-            .pointerInput(Unit) {
-                detectDown {
-                    hide()
-                }
+            .onPointerEvent(PointerEventType.Press, pass = PointerEventPass.Initial) {
```

Comment:
Why do we need `Initial` instead of the main pass here?

Reply:
I don't know, but it was in the code already, so I decided not to change it.